staging/src/k8s.io/client-go/dynamic/interface.go

```diff
@@ -29,15 +29,15 @@ type Interface interface {
 }
 
 type ResourceInterface interface {
-	Create(obj *unstructured.Unstructured, subresources ...string) (*unstructured.Unstructured, error)
-	Update(obj *unstructured.Unstructured, subresources ...string) (*unstructured.Unstructured, error)
+	Create(obj *unstructured.Unstructured, options metav1.CreateOptions, subresources ...string) (*unstructured.Unstructured, error)
+	Update(obj *unstructured.Unstructured, options metav1.UpdateOptions, subresources ...string) (*unstructured.Unstructured, error)
 	UpdateStatus(obj *unstructured.Unstructured) (*unstructured.Unstructured, error)
```

Comment:
Sorry, I missed this on my first pass.  This one should be able to pass updateoptions too.

Reply:
Good catch. I can't remember why I thought we wouldn't do this one ...